caf-ssl-dropwizard/pom.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2024 Open Text.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>com.github.cafapi.ssl</groupId>
+        <artifactId>caf-ssl</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>caf-ssl-dropwizard</artifactId>
+    <packaging>jar</packaging>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.dropwizard</groupId>
+            <artifactId>dropwizard-core</artifactId>
+            <exclusions>
+                <!-- Excluding this due to the following error:
+                     Found duplicate and different classes in [com.sun.activation:jakarta.activation:2.0.1, jakarta.activation:jakarta.activation-api:2.1.0]
+                 -->
+                <exclusion>
+                    <groupId>com.sun.activation</groupId>
+                    <artifactId>jakarta.activation</artifactId>
+                </exclusion>
```

Comment:
What's going on here?

Reply:
The dependency check was failing so I had to exclude one of them:

```
[INFO] Checking test classpath
[WARNING] Found duplicate and different classes in [com.sun.activation:jakarta.activation:2.0.1, jakarta.activation:jakarta.activation-api:2.1.0]:
[WARNING]   jakarta.activation.ActivationDataFlavor
[WARNING]   jakarta.activation.CommandMap
[WARNING]   jakarta.activation.DataHandler
[WARNING]   jakarta.activation.DataHandlerDataSource
[WARNING]   jakarta.activation.DataSourceDataContentHandler
[WARNING]   jakarta.activation.FileDataSource
[WARNING]   jakarta.activation.MailcapCommandMap
[WARNING]   jakarta.activation.MimeTypeParameterList
[WARNING]   jakarta.activation.MimetypesFileTypeMap
[WARNING]   jakarta.activation.ObjectDataContentHandler
[WARNING]   jakarta.activation.SecuritySupport
[WARNING] Found duplicate classes/resources in compile classpath.
[WARNING] Found duplicate and different classes in [com.sun.activation:jakarta.activation:2.0.1, jakarta.activation:jakarta.activation-api:2.1.0]:
[WARNING]   jakarta.activation.ActivationDataFlavor
[WARNING]   jakarta.activation.CommandMap
[WARNING]   jakarta.activation.DataHandler
[WARNING]   jakarta.activation.DataHandlerDataSource
[WARNING]   jakarta.activation.DataSourceDataContentHandler
[WARNING]   jakarta.activation.FileDataSource
[WARNING]   jakarta.activation.MailcapCommandMap
[WARNING]   jakarta.activation.MimeTypeParameterList
[WARNING]   jakarta.activation.MimetypesFileTypeMap
[WARNING]   jakarta.activation.ObjectDataContentHandler
[WARNING]   jakarta.activation.SecuritySupport
[WARNING] Found duplicate classes/resources in runtime classpath.
[WARNING] Found duplicate and different classes in [com.sun.activation:jakarta.activation:2.0.1, jakarta.activation:jakarta.activation-api:2.1.0]:
[WARNING]   jakarta.activation.ActivationDataFlavor
[WARNING]   jakarta.activation.CommandMap
[WARNING]   jakarta.activation.DataHandler
[WARNING]   jakarta.activation.DataHandlerDataSource
[WARNING]   jakarta.activation.DataSourceDataContentHandler
[WARNING]   jakarta.activation.FileDataSource
[WARNING]   jakarta.activation.MailcapCommandMap
[WARNING]   jakarta.activation.MimeTypeParameterList
[WARNING]   jakarta.activation.MimetypesFileTypeMap
[WARNING]   jakarta.activation.ObjectDataContentHandler
[WARNING]   jakarta.activation.SecuritySupport
[WARNING] Found duplicate classes/resources in test classpath.
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for caf-ssl 1.0.0-SNAPSHOT:
[INFO]
[INFO] caf-ssl ............................................ SUCCESS [  8.292 s]
[INFO] caf-ssl-dropwizard ................................. FAILURE [ 10.150 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  21.162 s
[INFO] Finished at: 2024-02-26T15:26:23Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.basepom.maven:duplicate-finder-maven-plugin:2.0.1:check (deepcheck-dependencies) on project caf-ssl-dropwizard: Found duplicate classes/resources! -> [Help 1]
```

![image](https://github.com/CAFapi/caf-ssl/assets/55391479/e75e9008-aeeb-447d-b4bb-e523fae4c784)

https://stackoverflow.com/a/77243009